build.gradle

```diff
@@ -47,6 +47,10 @@ ext {
     downloadJdkDir = "$buildDir/jdk"
 }
 
+def isLinux() {
+    return hasProperty( 'os' ) ? (os == 'linux') : false
```

Comment:
Just a comment. Mostly for @jsi 
This may introduce historical inconsistency.
generic (without jdk/jre) build used to be built with no `os` property. now generic is always built with `os` = `linux`